### PR TITLE
Bugfix/1768 bump docker version ci fix

### PR DESCRIPTION
**Pull request by sayf-eddine-scality**
**Component**:
check #1768 
check #1780 

**Context**: 

**Summary**:

**Acceptance criteria**: 

---

Closes: #1768 
Closes: #1780

### CODE REVIEW

File: eve/workers/pod-builder/pod.yaml [hunk 1]
```diff
@@ -28,14 +28,17 @@ spec:
...
+      image: docker:19.03.2-dind
       resources:
         requests:
           cpu: "3"
           memory: 2Gi
         limits:
           cpu: "3"
           memory: 2Gi
+      env:
+        - name: DOCKER_TLS_CERTDIR
```
Comment (slaperche-scality) [inline eve/workers/pod-builder/pod.yaml:40]: Should explain why this in the commit message (you only talk about bumping the version)

Comment (tcarmet) [reply eve/workers/pod-builder/pod.yaml:40]: I'm curious for this one as well :popcorn:

Comment (sayf-eddine-scality) [reply eve/workers/pod-builder/pod.yaml:40]: done

**Review by slaperche-scality — APPROVED** [review]

**Comment by NicolasT** [comment]
FWIW, ran into this a couple of times as well last night, so eager to see this land!

It'd be good to, ASAP, *not* require to 'build the previous version' in CI, of course...

**Pull request merged**